docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+
+In this case both the *destination1* and *destination2* clusters have been selected to offload the namespace `liqo-demo`.
+
+```bash
+kubectl get namespaceoffloadings offloading -n liqo-demo -o yaml
+```
+
+```yaml
+...
+status:
+  observedGeneration: 1
+  offloadingPhase: Ready
+  remoteNamespaceName: liqo-demo
+  remoteNamespacesConditions:
+    destination1-4591d5:
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: The remote cluster has been selected through the ClusterSelector
+        field
+      reason: ClusterSelected
+      status: "True"
+      type: OffloadingRequired
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: Namespace correctly offloaded to the remote cluster
+      reason: NamespaceCreated
+      status: "True"
+      type: Ready
+    destination2-9ea373:
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: The remote cluster has been selected through the ClusterSelector
+        field
+      reason: ClusterSelected
+      status: "True"
+      type: OffloadingRequired
+    - lastTransitionTime: "2023-01-22T21:26:11Z"
+      message: Namespace correctly offloaded to the remote cluster
+      reason: NamespaceCreated
+      status: "True"
+      type: Ready
+```
+
+Indeed, if you query for the namespaces in the *destination1* or *destination2* cluster, you should see the following output, confirming that the remote namespace has been correctly created by Liqo:
+
+```bash
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION1"
+kubectl get namespaces liqo-demo --kubeconfig="$KUBECONFIG_DESTINATION2"
+```
+
+Both the commands should return the following output:
+
+```text
+NAME        STATUS   AGE
+liqo-demo   Active   70s
+```
+
+## Deploy applications
+
+Now it's time to deploy the application.
+
+In order to create a replica of the application in every *destination* cluster is necessary to enforce these conditions.
+
+* The *deployment* resource must produce at least one *pod* for each *destination* cluster.
+* Each *destination* cluster must schedule at most one *pod* on its nodes.
+
+To get this result you can exploit these *kubernetes* features:
+
+* Set a number of replicas in the *deployment* which is equal to the number of *destination* clusters
+* Set *topologySpreadConstraints* inside the *deployment*'s template, which sets the *maxSkew* equal to **1**.
+
+The file `manifests/deployment.yaml` contains an example of a *deployment* which satisfies these conditions.
+Let's deploy it:
+
+```bash
+kubectl apply -f ./manifests/deployment.yaml -n liqo-demo
+```
+
+```{admonition} More replicas
+If the deployment uses a number of replicas which is higher than the number of *virtual nodes*, the pods will be scheduled respecting the *maxSkew* value, which guarantees that the difference between the maximum number of pods (scheduled on a single node) and the minimum will be **1**
```

Comment:
```suggestion
If the deployment uses a number of replicas which is higher than the number of *virtual nodes*, the pods will be scheduled respecting the *maxSkew* value, which guarantees that the difference between the maximum number of pods (scheduled on a single node) and the minimum will be **1**.
```
(honestly, I did not understand exactly what happens, but I have to say that is likely a *my* problem)

Reply:
I added a link to documentation about *maxSkew* to make it more clear. It is a bit tricky to understand.